media-sound/teamspeak-client/files/teamspeak-client-3.2.3-qt-5.12-symbol-reexports.patch

```diff
@@ -0,0 +1,141 @@
+diff --git a/Makefile b/Makefile
+new file mode 100644
+index 0000000..c036d5f
+--- /dev/null
++++ b/Makefile
+@@ -0,0 +1,2 @@
++overrides.so: qt-5.12-overrides.o override-version.map
++	$(CXX) $(LDFLAGS)  -shared -o $@ -Wl,--version-script override-version.map qt-5.12-overrides.cpp
```

Comment:
Looks good, but there is now a QA warning, which needs to be fixed:

```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /opt/teamspeak3-client/overrides.so
```

You can fix this, by adding ```$(CFLAGS)``` to ```$(CXX) $(CFLAGS) $(LDFLAGS)``` here.

Reply:
This looks weird. This line describes the link stage; the compile stage is performed by the built-in `make` rule that takes into account CFLAGS on its own. Also, I've run the build and haven't seen this QA (or do I need some additional flags to see QA warnings?). Are the CFLAGS required at the link stage?

Reply:
Did you enabled those QA warning? As default, you won't see them. To get them enabled, you must set ```-frecord-gcc-switches``` to ```CFLAGS```, ```CXXFLAGS```, ```FCFLAGS``` and ```FFLAGS```

```
	# Check for files built without respecting *FLAGS. Note that
	# -frecord-gcc-switches must be in all *FLAGS variables, in
	# order to avoid false positive results here.
	# NOTE: This check must execute before prepall/prepstrip, since
	# prepstrip strips the .GCC.command.line sections.
	if [[ "${CFLAGS}" == *-frecord-gcc-switches* ]] && \
	[[ "${CXXFLAGS}" == *-frecord-gcc-switches* ]] && \
	[[ "${FFLAGS}" == *-frecord-gcc-switches* ]] && \
	[[ "${FCFLAGS}" == *-frecord-gcc-switches* ]] ; then
```

Reply:
Ok, this was a stupid error on my side (.cpp as input instead of .o), fixed.